config/locales/diaspora/en.yml

```diff
@@ -921,68 +921,68 @@ en:
         policy: "See the application's privacy policy"
       scopes:
         openid:
-          name: "basic profile"
-          description: "This grants read access to basic profile information to the application."
+          name: "Basic profile information"
+          description: "This grants read-only access to your basic profile information."
         sub:
-          name: "sub"
-          description: "This grants sub permissions to the application."
+          name: "Unique identifier"
+          description: "This grants read-only access to your unique identifier."
         name:
-          name: "name"
-          description: "This grants read access to name data to the application."
+          name: "Full name"
+          description: "This grants read-only access to your full name."
         nickname:
-          name: "nickname"
-          description: "This grants read access to nickname data to the application."
+          name: "Username"
+          description: "This grants read-only access to your username."
         picture:
-          name: "picture"
-          description: "This grants read access to user's profile picture data to the application."
+          name: "Profile picture"
+          description: "This grants read-only access to your profile picture."
         'contacts:read':
-          name: "contacts:read"
-          description: "This grants read permissions to contacts and related data (like aspects) to the application."
+          name: "Contacts (Read-only)"
+          description: "This grants read-only access to contacts and related data (like aspects)."
```

Comment:
A minor style point: for clarity, I would use 'such as' rather than 'like' in 'related data (like aspects)'. But 'like' in this context is very common, and there's nothing wrong with it.

Reply:
Fair suggestion, I'll change that.